client/app/components/Parameters.jsx

```diff
@@ -146,15 +150,17 @@ export default class Parameters extends React.Component {
 
   render() {
     const { parameters } = this.state;
-    const { editable } = this.props;
+    const { sortable, appendSortableToParent } = this.props;
     const dirtyParamCount = size(filter(parameters, "hasPendingValue"));
+
     return (
       <SortableContainer
-        disabled={!editable}
+        disabled={!sortable}
         axis="xy"
         useDragHandle
         lockToContainerEdges
         helperClass="parameter-dragged"
+        helperContainer={containerEl => (appendSortableToParent ? containerEl : document.body)}
```

Comment:
What exactly does this fix?

Reply:
By standard React Sortable HOC will append the draggable to the document body. The creator of our own component decided to add a container to it, which wasn't clear to me why, until I removed it and noticed minor undesirable UI changes. I brought it back and added this as an alternative to only change what needs to be.

In the case of widgets, it's necessary to append them to a component in an upper scope than that of their containers, as they are positioned using transforms, which mess with the final position of the draggable. I tried to workaround them, but it got too complex and this seemed clearer.